### ISSUE TITLE
Popup every access on Italy

### ISSUE THREAD

**Issue by menteora** [ugorsahin/ChatGPT_Automation#33]
Hi, there is a way to bypass this banner on Italy?

```
We’re pleased to resume offering ChatGPT in Italy. To continue on ChatGPT, please confirm that you are 18+ or are 13+ and have consent from your parent or guardian to use ChatGPT.
For information about how we collect and use personal data, please see our Privacy policy. For information about how we develop and train ChatGPT, please see this help center article.
```
The question is asked at every login and doesn't allow you to continue

Thankyou

**Comment by ugorsahin** [comment]
This is a quite interesting situation. I will try to replicate the popup with VPN and find a solution.

**Comment by ugorsahin** [comment]
I tried to connect through Italy, but couldn't view any popup on my end. Can you share a screenshot to show me? Perhaps it is a popup that should be clicked only once.

**Comment by menteora** [comment]
You're right, I just re-ran the tests, and now the popup no longer appears. I'm attaching the screenshot (searched on internet) for your reference, but we can close the issue, and thank you very much for your support.

![1682745074623](https://github.com/ugorsahin/ChatGPT_Automation/assets/3657745/be537221-2579-4af2-8ba9-4ad122b59f6c)
